values.yaml

```diff
@@ -321,6 +321,13 @@ server:
   # ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
   priorityClassName: ""
 
+  # Extra labels to attach to the server pods.
+  # This should be a multi-line YAML string.
+  # Example:
+  #   podLabels: |
+  #     "label-key": "label-value"
+  podLabels: null
```

Comment:
Can this be renamed to `extraLabels`? That seems to be aligned with how we have addressed similar constructs in the values file. see: extraConfig, extraVolumes.

Additionally, can you update the example to demonstrate setting multiple labels? It should help other users when they are trying to set this up

Reply:
Done :)